### PR TITLE
Remove warning about rc_io support if not APM

### PR DESCRIPTION
**Pull request by AndreasAntener**
**untested** @scott-eddy

### CODE REVIEW

**Comment by AndreasAntener** [comment]
Support in PX4 for implemented a while back: https://github.com/PX4/Firmware/blob/9647f0622c22b9b1dd3603970a75bb3006a056e7/src/modules/mavlink/mavlink_receiver.cpp#L1045

File: mavros/src/plugins/rc_io.cpp [hunk 1]
```diff
@@ -213,9 +213,6 @@ class RCIOPlugin : public MavRosPlugin {
 	}
 
 	void override_cb(const mavros_msgs::OverrideRCIn::ConstPtr req) {
-		if (!uas->is_ardupilotmega())
```
Comment (vooon) [inline mavros/src/plugins/rc_io.cpp]: I prefer change condition to:
```cpp
if (!uas->is_ardupilotmega() && !uas->is_px4())
```

Comment (AndreasAntener) [inline mavros/src/plugins/rc_io.cpp]: Makes sense

**Comment by vooon** [comment]
Thank you!

**Pull request merged**